project_template/tasks.py

```diff
@@ -163,6 +163,27 @@ class TaskOwnedIncomeFromPensionsTable(CountTableRowsTask):
     child_class = TaskOwnedIncomeFromPensionsRowEntry
 
 
+class TaskOwnedGoodsOrServicesPerOwnerRowEntry(DigitalizationTask):
+    task_form = forms.TranscribeOwnedGoodsOrServicesPerOwnerSingleRowTable
+    template_name = "task/owned_goods_or_services_per_owner.html"
```

Comment:
The folder where we keep the templates is called "tasks", please check the path: https://github.com/code4romania/catpol-declaratii/tree/master/project_template/templates/tasks

Also, the html file you reference here is missing from the pull request and the server will not start without it.
Please check the above path, look at any one of the html files and create a similar one, add it to the pull request.